lib/gears/geometry.lua

```diff
@@ -11,6 +11,10 @@
 ---------------------------------------------------------------------------
 local math = math
 
+local pow = math.pow or function(a,b) -- lua 5.3 compat
+    return a^b
```

Comment:
Is this necessary? Is there a Lua version without `^`? I tried with Lua 5.1 and LuaJIT2